### ISSUE TITLE
golang: add 'scw run'

### ISSUE THREAD

**Issue by moul** [scaleway/scaleway-cli#39]

**Comment by moul** [comment]
Fixed with 54b6e1ce5bc64b23d857ebdb0bbdd1f49f008001

**Comment by moul** [comment]
```console
➜  scaleway-cli-go git:(go) ✗ clear; make && scw -D run alpine sh
gofmt -w ./.
go test -i ./.
go build -o scw ./.
DEBU[0000] Creating a new server
DEBU[0000] POST https://api.scaleway.com/servers payload={"name":"insane-torvalds","image":"1b9702de-a665-497e-babb-36674843d94d","bootscript":null,"tags":[],"organization":"6d6b64e5-6bad-4cc6-b7ef-2030884c3e11"}
DEBU[0000] Created server: cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0000] Starting server
DEBU[0000] POST https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c/action payload={"action":"poweron"}
DEBU[0001] Server is booting
DEBU[0001] Waiting for server to be ready
DEBU[0001] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0003] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0004] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0005] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0007] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0009] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0010] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0012] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0013] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0015] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0016] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0017] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0019] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0021] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0024] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0025] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0027] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0029] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0030] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0032] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0033] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0035] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0037] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0038] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0040] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0041] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0043] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0044] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0046] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0047] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0049] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0050] GET https://api.scaleway.com/servers/cd22f13d-ea80-4689-9f1e-159e2594028c
DEBU[0054] Server is ready: 212.47.231.72
DEBU[0054] Executing command
DEBU[0054] Executing: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root 212.47.231.72 -t -- /bin/sh -e -x -c "sh"
Warning: Permanently added '212.47.231.72' (RSA) to the list of known hosts.
+ sh
~ # ls -la
total 20
drwx------    3 root     root          4096 Jun  1 16:05 .
drwxr-xr-x   18 root     root          4096 Jun  1 16:05 ..
-rw-------    1 root     root             7 Jun  1 16:05 .ash_history
-rw-rw-r--    1 root     root          1240 Apr  1 15:19 .s3cfg.sample
drwx------    2 root     root          4096 Jun  1 16:05 .ssh
~ # Shared connection to 212.47.231.72 closed.
DEBU[0068] Executed command: <nil>
```
